src/components/Header.js

```diff
@@ -53,7 +58,11 @@ export default function Header() {
                 <Link to="/cart">
                   <i className="fas fa-shopping-cart"></i>
 
-                  {cartItemsCount>0&&<span className="status-badge badge-wth-number round-badge">{cartItemsCount}</span>}
```

Comment:
instead of checking here again > 0 , you can do the same thing at line no 13 and 14.

Like this :-
  const cartItemsCount = cartData.length > 0;
  const wishlistCount = wishlistData.length > 0;